### DISCUSSION TITLE
屏蔽黑名单up组件如何使用

### DISCUSSION THREAD

**Discussion by LunaGlaze** [the1812/Bilibili-Evolved#4510] [提问 / Q&A]
根据组件说明在首页进行了配置，精确匹配和正则匹配皆已经设置，但相关UP依旧出现在首页，故来求证是否为本人使用方式错误或理解错误：

![PD~JBPK7J %LQXL031C49F](https://github.com/the1812/Bilibili-Evolved/assets/120086484/51b0346a-919d-4c3d-b494-50d522dd7b38)
![~6 $9VHY3UBY_XK4`1@@{JC](https://github.com/the1812/Bilibili-Evolved/assets/120086484/8db84d8f-2d91-4956-9df3-eb152e70c14f)
正则亦尝试过直接输入关键词文本未添加符号，最终结果相同，被屏蔽UP仍然出现在首页
![61EMSQ$X~GUOAW1YZ1NEBTD](https://github.com/the1812/Bilibili-Evolved/assets/120086484/8f77970f-b872-43f9-93d2-ba88ee4d6097)

本体版本：v2.8.5-71-g10300c16b
组件版本未知如何查询，但已经检查更新

**Comment by the1812** [comment]
@snowraincloud 可能首页更新了导致组件失效了

**Comment by snowraincloud** [comment]
已提pr修复
